L/Libtask/build_tarballs.jl

```diff
@@ -43,15 +43,8 @@ install_license LICENSE.md
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
-
-# skip i686 musl builds (not supported by libjulia_jll)
-filter!(p -> !(Sys.islinux(p) && libc(p) == "musl" && arch(p) == "i686"), platforms)
-
-# skip PowerPC builds in Julia 1.3 (not supported by libjulia_jll)
-if julia_version < v"1.4"
-    filter!(p -> !(Sys.islinux(p) && arch(p) == "powerpc64le"), platforms)
-end
+include("../../L/libjulia/common.jl")
+platforms = libjulia_platforms(julia_version)
```

Comment:
It doesn't seem like this needs to expand the C++ string ABI (see also the [latest release](https://github.com/JuliaBinaryWrappers/Libtask_jll.jl/releases/tag/Libtask-v0.4.2%2B0)).  It's probably better to have `libjulia_platforms` return the list of compiler-ABI-agnostic platforms, and explicitly expand the C++ string ABI where necessary.

Reply:
Good point

Reply:
Updated.

Reply:
@giordano OK to merge now?